components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/SQLConstants.java

```diff
@@ -69,6 +69,18 @@ public class SQLConstants {
             SQLPlaceholders.DB_SCHEMA_COLUMN_NAME_PARENT_ID + ";, :" + SQLPlaceholders.DB_SCHEMA_COLUMN_NAME_ID +
             ";, 1 FROM dual ) SELECT * FROM OH";
 
+    public static final String INSERT_ROOT_ORGANIZATION = "INSERT INTO UM_ORG (UM_ID, UM_ORG_NAME, " +
```

Comment:
why we need a different SQL constant for this.
Can't we use "INSERT_ORGANIZATION" ?
Only difference I see is we are not setting UM_ORG_DESCRIPTION here. anyway that's an optional field

Reply:
Here only parentID is not set which is a null value. Added a separate query as passing null for prepared statement might cause issues.
SQLPlaceholders.DB_SCHEMA_COLUMN_NAME_PARENT_ID ;